root/etc/httpd/conf.d/ejabberd.conf

```diff
@@ -9,6 +9,6 @@ SSLProxyCheckPeerCN off
 SSLProxyCheckPeerName off
 SSLProxyCheckPeerExpire off
 
-ProxyPass /http-bind https://127.0.0.1:5280/http-bind
-ProxyPassReverse /http-bind https://127.0.0.1:5280/http-bind
+ProxyPass /bosh https://127.0.0.1:5280/bosh
+ProxyPassReverse /bosh https://127.0.0.1:5280/bosh
```

Comment:
```suggestion
ProxyPass /http-bind https://127.0.0.1:5280/bosh
ProxyPassReverse /http-bind https://127.0.0.1:5280/bosh
```

Reply:
I did two proxypass : bosh and http-bind